src/cephadm/cephadm

```diff
@@ -494,6 +494,91 @@ class CephIscsi(object):
 
 ##################################
 
+class HAproxy(object):
```

Comment:
do we want this to be named a more specific name for rgw like  'rgw_haproxy' or is this generic enough it could be used to HA a different service

Reply:
I prefer to use for the systemd units and the daemons running in each hosr the right name. In this case is just an HAproxy container, so the name is correct. 
The same will happen with keealived, you will need to run a "keepalived" daemon not a "rgw_keepalived" daemon

Probably in the future we will add more things under the "haproxy+keepalived" umbrella. The  only thing we will need to do is to change the configuration files, but the running daemons allways will be "haproxy" and "keepalived"